examples/nextjs-with-typescript/components/SimpleVideoPlayer.tsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-console */
```

Comment:
This is a SimpleVideoPlayer that use hls.js and adds Mux Video DRM + playback token support.